USAGE.md

```diff
@@ -230,3 +230,26 @@ MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEjCxhhvb1KmIfe1J2ceT25kHepstb
 IDYuTA0U1ri4F0CXXazLiftzGlyfse1No4orr8w1ZIchQ8TJlyCSaSuR0Q==
 -----END PUBLIC KEY-----
 ```
+
+# Experimental Features
+
+## Verify a signature was added to the transparency log
+There are two options for verifying a cosign signature was added to a transparency log:
+1. Check the log to make sure the entry exists in the log
+2. If it exists, use the `bundle` annotation on a cosign signature to verify an element was added to the log without hitting the log
+
+The cosign `bundle` annotation contains a Signed Entry Timestamp (SET), which is conceptually similar to an SCT in a Web PKI system.
+The SET is a signed inclusion promise provided by the transparency log, which acts as a guarantee by the log that an element has been included in it.
+The SET can be verified with the logs public key and used to prove that an element is in the log without actually checking the log itself.
+
+For more details on how the `bundle` annotation is formatted, see the cosign [spec](SPEC.md).
+
+To verify the `bundle` annotation, follow these steps:
+1. Create a rekor [LogEntryAnon](https://github.com/sigstore/rekor/blob/main/pkg/generated/models/log_entry.go#L84) type, and initialize with the following values from the `bundle`:
+* IntegratedTime
```

Comment:
Could we store this entire thing in the bundle somewhere to make it easier?

Reply:
I added a `Payload` struct in the bundle to make this easier -- the only thing is it isn't backwards compatible with previous bundles.

We could either:
1) Just change the spec, and people verifying the old version of the bundle will have to hit the tlog 
1) Leave in support for both types of bundles for a couple versions

I'm fine with either, right now the PR is at option 1!

Reply:
Yea that's fine!